### ISSUE TITLE
Missing java-snapshot-testing-core in maven central?

### ISSUE THREAD

**Issue by ashatch** [origin-energy/java-snapshot-testing#24]
I'm likely doing something dumb, but a search for the [groupId in maven central](https://search.maven.org/search?q=io.github.origin-energy) doesn't include `java-snapshot-testing-core`.

Am I doing something wrong?

```xml
        <dependency>
            <groupId>io.github.origin-energy</groupId>
            <artifactId>java-snapshot-testing-core</artifactId>
            <version>2.0.0-Beta5</version>
            <scope>test</scope>
        </dependency>
        <dependency>
            <groupId>io.github.origin-energy</groupId>
            <artifactId>java-snapshot-testing-junit5</artifactId>
            <version>2.0.0-Beta5</version>
            <scope>test</scope>
        </dependency>
```
`java-snapshot-testing-junit5` resolves just fine, but `java-snapshot-testing-core` doesn't.

Thanks in advance. 👍

**Comment by jackmatt2** [comment]
Yeah you're right - it's bundled into each framework implementation

```
implementation project(':java-snapshot-testing-core')
```

I did this originally to make it easier for people so they didn't need to include both core and framework and then make sure they eep the two dependencies in sync.

Probably a mistake for people that want to implement their own framework.  Can probably look at publishing separately too.

**Comment by jackmatt2** [comment]
I think I know the issue you are having now - Looks like I didn't bundle it correctly.  Will need to get it fixed and do a new release.

**Comment by jackmatt2** [comment]
This should be fixed as of `2.0.0-Beta6` - maven sync can take some time.
